content/blog/multi-lang-components/index.md

```diff
@@ -18,27 +18,35 @@ Pulumi's infrastructure as code tooling combines the programming languages and t
 
-Now, with multi-language components, anyone can author a Pulumi component in the language they prefer and make it available
-across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
-and made it available in all four Pulumi languages: TypeScript, Python, .NET, and Go. Regardless of the language you choose, you can
-[manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the EKS package]. Check out our Modern Infrastructure
-Wednesday video to see it in action:
+Now, you can use the [EKS package]–previously only available for TypeScript–in all four Pulumi languages: TypeScript, Python, .NET,
+and Go. Regardless of the language you choose, you can [manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the
+EKS package]. Check out our Modern Infrastructure Wednesday video to see it in action:
 
 {{< youtube "gxLyAr0lUg0?rel=0" >}}
 
 <!--more-->
 
-EKS in all languages is just the start of the multi-language components story. Multi-language components enable all organizations to choose
-the right programming language, or languages, for each team and each product. Your infrastructure or platform teams can
-write components in Go and share them with a webapp team using TypeScript, or write in .NET and share with a machine learning team
-using Python or any other combination that suits your business. Similarly, any member of the worldwide Pulumi community can author
-a component in the language they're most comfortable with and share it with all Pulumi users.
+EKS in all languages is made possible by our ongoing work to enable "[multi-language components]". With multi-language components,
+any member of the worldwide Pulumi community will be able to author a Pulumi component in the language they're most comfortable
+with and share it with all Pulumi users: even those that use a different language than the component was authored in. Multi-language
+components take care of the conversion from one language to other Pulumi languages, so component developers don't need detailed
+knowledge of the other languages.Multi-language components will also enable organizations to write Pulumi components in the right
+programming language, or languages, for each team and each product. An infrastructure or platform teams will be able to write
+components in Go and share them with a webapp team using TypeScript, or write in .NET and share with a machine learning team using
+Python, or any other combination that suits your business.
 
-The next step in the story is a seamless authoring experience that enables component creators to write, build, and publish components
-without detailed knowledge of the various Pulumi languages. We're excited for you to try it soon - check back here for updates!
+### Next steps
+
+The next step in the story is a public preview of a seamless authoring experience that enables component creators to write, build,
```

Comment:
```suggestion
We are hard at work to release the next phase of this feature into the public preview: a seamless authoring experience that enables component creators to write, build,
```

Reply:
"that will enable component creators to write, build,..."

Reply:
```suggestion
The next step in the story is a public preview of a seamless authoring experience that will enable component creators to write, build,
```